app/src/main/res/drawable/ic_arrowhead_next.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We have [`mozac_ic_arrowhead_right`](https://github.com/mozilla-mobile/android-components/blob/main/components/ui/icons/src/main/res/drawable/mozac_ic_arrowhead_right.xml) in AC already.

Reply:
Ok, I will add ui-icons to dependencies.